amundsen_application/static/js/components/common/ResourceListItem/styles.scss

```diff
@@ -4,34 +4,54 @@
   color: $text-dark;
   display: flex;
   flex-direction: row;
-  height: 78px;
-  padding: 16px 8px;
+  height: 96px;
+  padding: 24px 24px;
   text-decoration: none;
 
-  img.icon,
+  img.icon.start-icon,
   .sb-avatar {
-    margin: auto 0;
+    margin: auto 16px auto 0px;
+  }
+  
+  .icon-right {
+    margin: auto 0 auto auto;
+    opacity: 0;
   }
 
-  &:hover img.icon {
-    background-color: $brand-color-4;
+  &:hover .icon-right,
+  &:focus .icon-right {
+    opacity: 1;
   }
 
-  .content {
-    // TODO - Replace width and min-width with flex: 1
-    width: 100%;
-    min-width: 0; /* Needed to support `white-space: nowrap` */
+  .resource-info {
+    display: flex;
+    flex: 7;
+    min-width: 0px;
 
-    .resource-name .truncated {
-      display: inline-block;
-      max-width: calc(100% - 32px);
+    .resource-name {
+      display: flex;
     }
   }
-}
 
+  .resource-type {
+    flex: 2;
```

Comment:
Can the resource-type have `flex: 0`? I don't see the need to expand this like the resource-info or resource-badges.

Reply:
How about we take this up in our design QA sync? I checked with design today about how much space each section should have, and our layout specifications for this page were intended to have a 7/2/3 split. 

Perhaps there's something Design is planning for the future that we don't know of, or perhaps the decision came from some other design concept that they are adhering to.